src/blister.c

```diff
@@ -1138,6 +1138,8 @@ Int IsBlistConv (
 **  COUNT_TRUES_BLOCK( <var> ) macro which replaces a block of bits in <var> 
 **  by the number of ones it contains. It will fail horribly if <var> is not 
 **  a variable.
+**
+**  Switched COUNT_TRUES_BLOCK to an inline function that returns the count
```

Comment:
What is the purpose of the comment, and the ones right before it? Seems like a partially revision history? But that belongs in the commit logs, not a code comment....